bodhi/server/ffmarkdown.py

```diff
@@ -149,4 +149,4 @@ def extendMarkdown(self, md, md_globals):
         """
         md.inlinePatterns.add('mention', MentionPattern(MENTION_RE, md), '_end')
         md.inlinePatterns.add('bugzilla', BugzillaPattern(BUGZILLA_RE, md), '_end')
-        md.postprocessors.add('surrond', SurroundProcessor(md), '_end')
+        md.postprocessors.add('surround', SurroundProcessor(md), '_end')
```

Comment:
It's possible this is intentionally matching `surrond`. If that's the case, it'd be a good idea to add a comment here.

Reply:
Since this is likely to require discussion / to simplify reviews, I've moved it to #3134